translation/en/RedirectedResources/assets/abdata/adv/scenario/c06/20/1003/translation.txt

```diff
@@ -4,35 +4,35 @@
-//「それは……お気持ちだけ、受け取らせていただきます」=
-//「えっ、ど、どういうこと？」=
-//「魅力的な殿方は数多くの女性に求められるものだからでございます。私に全てを捧げてしまっては、他の女性に愛を注げなくなってしまいますよ」=
-//「私と貴方が結婚して、たとえば世間一般に浮気と称されることを貴方がされても、私は変わらず貴方に尽くします」=
-//「女がただ一人の殿方を心に決めるとは、そういうことです」=
-//「そ、そんなの、[H名]が可哀想じゃないか」=
-//「今の時代に照らし合わせればそうなのかもしれません。ですが、私は古い女として育てられ、自分でもそれを良しとしております」=
-//「女は殿方を喜ばせ、妻は夫を立てる。男女平等が声高に謳われ、ともすれば女性の方が優位に立っている世の中で、私だけは大和撫子として咲き誇りたいのです」=
-//「[H名]……そんなこと言われたら、ますます[H名]だけ愛したくなっちゃうよ」=
-//「もちろん、私だけを愛していただけるのでしたら、それに勝る幸せはございません」=
-//「ですが、先ほどの言葉は裏を返せば、他の女性と比べられても貴方に選ばれる自信がある、ということでございます。私にはそれだけ女を磨いて来た自負がございます」=
-//「うん……[H名]はとっても女の子らしい女の子だし、男心を一番わかってると思うよ」=
-//「そう仰っていただけるなら、安心して他の女性のもとへ送り出すことができます。こんな私で宜しければ、誓いの口付けをしていただけますか？」=
-//「[H名]……もちろん。気苦労ばっかりかけるかもしれないけど、一番は[H名]だって、ずっと証明し続けるからね」=
-//「ん……」=
-//「んんっ……」=
+「[H名]さん、あなたはこの男性とお付き合いし、より親密な関係になろうとしております」=“Miss [H名], You have been going out with this man for a while now. And now you are proceeding towards an even closer relationship.”
+「それでは、二人の愛の証として、誓いのキスをお願いします」=“Then, without further ado, as the proof of your love, seal these vows with a kiss.”
+「……[H名]、さっきの話の続きなんだけど……俺も、[H名]との結婚を前提にして付き合いたいと思う」=“...[H名], picking up where we left off before... I want to, continue dating you too, with marriage in mind.”
+「そうじゃないと、いつも尽くしてくれる[H名]に対して、フェアじゃないから。俺は本気で[H名]のことが好きなんだってこと、誓いたい」=“It wouldn't be fair to you, if I don't. With how much you serve me, and look after me. I want to take an oath here, full of my love for you.”
+「それが貴方の本心なのですね。それなら私も、もう遠慮はいたしません。決して長くはない一生涯、貴方の為に捧げ、尽くすことを誓います」=“That's the answer you've arrived at, is it? In that case, I won't hold back anymore either. I vow to relinquish this short life of mine to you, to live my days serving you.”
+「俺も[H名]に全てを捧げるよ」=“I relinquish my everything to you as well, [H名].”
+「それは……お気持ちだけ、受け取らせていただきます」=“That's... I will happily, accept just the feelings behind that.”
+「えっ、ど、どういうこと？」=“Eh--, w-what do you mean?”
+「魅力的な殿方は数多くの女性に求められるものだからでございます。私に全てを捧げてしまっては、他の女性に愛を注げなくなってしまいますよ」=“An attractive, charming man is chased by women from all over. If you relinquish your all to me, you won't be able to give your love to any other woman.”
+「私と貴方が結婚して、たとえば世間一般に浮気と称されることを貴方がされても、私は変わらず貴方に尽くします」=“Even if you do the act known as 'cheating' to others, after we get married, I would keep serving you, regardless.”
+「女がただ一人の殿方を心に決めるとは、そういうことです」=“That is what it means, for a woman to set her heart on one man.”
+「そ、そんなの、[H名]が可哀想じゃないか」=“T-That's not fair to you. How could I do that?”
+「今の時代に照らし合わせればそうなのかもしれません。ですが、私は古い女として育てられ、自分でもそれを良しとしております」=“Perhaps so, through a modern viewpoint. However, I was raised in a traditional family with a traditional one, so even I find no problems with such a thing.”
+「女は殿方を喜ばせ、妻は夫を立てる。男女平等が声高に謳われ、ともすれば女性の方が優位に立っている世の中で、私だけは大和撫子として咲き誇りたいのです」=“A woman lives to make her man happy, a wife to support her husband. In a world where the ideals of gender equality are shouted from the rooftops, where more often than not women end up being the dominant ones, I alone want to bloom as a Yamato Nadeshiko.”
```

Comment:
I guess this is consistent with how the personality name is translated, but... maybe `as a traditional Japanese woman.`